tldw_Server_API/app/core/Slides/visual_style_packs.py

```
@@ -0,0 +1,268 @@
+    """Merge pack defaults with optional style-specific settings overrides."""
+
+    pack = _VISUAL_STYLE_PACKS_BY_ID.get(pack_id)
+    if pack is None:
+        return dict(settings_overrides or {})
+    resolved = dict(pack.default_resolved_settings)
+    if settings_overrides:
+        resolved.update(settings_overrides)
+    return resolved
+
+
+@lru_cache(maxsize=None)
```

Ruff (0.15.7)

[warning] Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)